srcalc/src/test/java/gov/va/med/srcalc/vista/vistalink/MockVistaLinkConnection.java

```diff
@@ -109,7 +109,7 @@ else if (request.getRpcName().equals(RemoteProcedure.GET_PATIENT.getProcedureNam
         {
             // RpcResponse is very hard to simulate. Use Mockito.
             RpcResponse response = mock(RpcResponse.class);
-            when(response.getResults()).thenReturn(PATIENT_DATA + "\n");
+            when(response.getResults()).thenReturn(PATIENT_DATA);
```

Comment:
Why did this change? I think the newline was there intentionally (to exactly simulate VistA).